test/hotspot/jtreg/ProblemList.txt

```diff
@@ -101,12 +101,12 @@ runtime/InvocationTests/invokevirtualTests.java#current-int 8271125 generic-all
 runtime/InvocationTests/invokevirtualTests.java#current-comp 8271125 generic-all
 runtime/InvocationTests/invokevirtualTests.java#old-int 8271125 generic-all
 runtime/jni/terminatedThread/TestTerminatedThread.java 8219652 aix-ppc64
-runtime/os/TestTracePageSizes.java#no-options 8267460 linux-aarch64
+runtime/os/TestTracePageSizes.java#no-options               8267460 linux-aarch64
```

Comment:
I don't think the whitespace changes to align fields should be made.  That's not the style of this file.  And I think it shouldn't be the style, since maintaining the alignment will frequently introduce otherwise uninteresting diffs.

Reply:
I agree with Kim.

Reply:
OK. I'll change this, though I think it makes it much harder to see what tests are excluded by which bug.